core/src/test/java/org/elasticsearch/search/aggregations/bucket/DateHistogramIT.java

```diff
@@ -107,8 +106,7 @@ private IndexRequestBuilder indexDoc(int month, int day, int value) throws Excep
 
     @Override
     public void setupSuiteScopeCluster() throws Exception {
-        assertAcked(prepareCreate("idx").addMapping("type", "_timestamp", "enabled=true"));
-        createIndex("idx_unmapped");
+        createIndex("idx", "idx_unmapped");
```

Comment:
I see now. `idx` gets some docs which automatically sets up some mappings but `idx_unmapped` is empty.